### PR TITLE
add rcp feature to the updatesite

### LINKED ISSUE #879: Please, export the repository for the rcp-designer also.
We are in the unfortunate position that we reuse the _org.eclipse.birt.report.designer.ui.rcp.DesignerApplication_ in our internal Report-Designer so we depend on the bundel _org.eclipse.birt.report.designer.ui.rcp_ which is not part of the normal update site: https://download.eclipse.org/birt/update-site/4.9.0/
The Designer product is currently only available as product export in https://download.eclipse.org/birt/downloads/drops/R-R1-4.9.0-202203161719 
Could you bring the corresponding updatesites also online? (birt-packages/birt-report-rcp)

Thanks.

### PR DESCRIPTION
**Pull request by UrsZeidler**
@wimjongman I think this should be sufficient. It's not really a contribution.
It adds the rcp feature to the updatesite. This should fix #879

### CODE REVIEW

File: build/org.eclipse.birt.p2updatesite/category.xml [hunk 1]
```diff
@@ -68,10 +68,18 @@
...
+   <feature id="org.eclipse.birt.rcp">
+      <category name="BirtRcp"/>
+   </feature>
    <category-def name="Birt" label="BIRT Reporting SDK"/>
    <category-def name="Chart" label="BIRT Charting SDK"/>
    <category-def name="OSGi" label="BIRT Engine OSGi Runtime SDK"/>
    <category-def name="wtpIntegration" label="BIRT WTP Integration SDK"/>
    <category-def name="BirtTest" label="Birt Unit Test"/>
    <category-def name="BirtNL" label="BIRT Language Package"/>
+   <category-def name="BirtRcp" label="BIRT RCP">
```
Comment (wimjongman) [inline build/org.eclipse.birt.p2updatesite/category.xml:80]: Urs, the label should also be clear about 'not install this'

**Review by wimjongman — CHANGES_REQUESTED** [review]

**Comment by UrsZeidler** [comment]
I have update my -very old eclipse account- https://accounts.eclipse.org/users/uzeidler2rv and filled out the ECA Form and it shows the green check mark in for the ECA.
Does the email of the eclipse account need to match the email of the github account?
I will add the notice to the category label.

greetings.

**Review by hvbtup — CHANGES_REQUESTED** [review]
Pls fix the typo: "bundel" should be "bundle"

**Review by hvbtup — APPROVED** [review]
The typo is fixed. 
Note: I can't say anything about the commit itself.

**Review by wimjongman — APPROVED** [review]

**Pull request merged**